src/replication.c

```diff
@@ -1420,6 +1410,14 @@ void replconfCommand(client *c) {
             } else {
                 c->flags &= ~CLIENT_REPL_MAIN_CHANNEL;
             }
+        } else if (!strcasecmp(c->argv[j]->ptr, "identify")) {
+            /* REPLCONF identify <client-id> is used to identify the current replica main connection with existing
+             * rdb-connection with the given id. */
```

Comment:
i think the term "identify" it too broad.
how about `REPLCONF set-rdb-conn-id <id>`?
also, please update the diagrams (ascii and image) with this step.

p.s. maybe we use `REPLCONF main-conn <id>` (instead sending two commands, one of them carrying a boolean)?
i.e. we'll need to postpone the `main-conn` till we know the id?
not sure if this could work, haven't looked into the details.

Reply:
I agree, I'll rename the subcommand and will update the diagrams today. 
`REPLCONF main-conn` should be called before the first PSYNC request (before the replica even knows it has to get the entire snapshot). Otherwise after PSYNC fails, the master will start to send the RDB using the main connection immediately.